packages/react-components/react-motion/stories/src/Tokens/Cards.tsx

```diff
@@ -0,0 +1,266 @@
+import * as React from 'react';
+import { Divider, makeStyles, motionTokens, tokens, Switch } from '@fluentui/react-components';
+
+const useClasses = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  controls: {
+    alignSelf: 'end',
+    backgroundColor: tokens.colorNeutralBackground1,
+    borderRadius: tokens.borderRadiusMedium,
+    borderBottomLeftRadius: 0,
+    borderBottomRightRadius: 0,
+    padding: '10px 20px',
+    border: `2px solid ${tokens.colorNeutralStroke1}`,
+    borderBottom: 'none',
+    width: 'fit-content',
+  },
+  card: {
+    backgroundColor: tokens.colorNeutralBackground1,
+    borderRadius: tokens.borderRadiusMedium,
+    borderTopRightRadius: 0,
+    padding: '30px',
+    display: 'grid',
+    gridTemplateColumns: 'repeat(auto-fill, minmax(250px, 1fr))',
+    gap: '30px 50px',
+    alignItems: 'center',
+    border: `2px solid ${tokens.colorNeutralStroke1}`,
+  },
+});
+
+const useCardClasses = makeStyles({
+  container: {
+    display: 'grid',
+    gridTemplate: `
+        "graph point"
+        "title ." auto / min-content var(--point-size)
+    `,
+    gap: '10px',
+
+    '--container-size': '250px',
+    '--point-size': '20px',
+  },
+  point: {
+    gridArea: 'point',
+
+    width: 'var(--point-size)',
+    height: 'var(--point-size)',
+
+    backgroundColor: tokens.colorBrandBackground,
+    clipPath: 'polygon(25% 0%, 100% 0%, 100% 100%, 25% 100%, 0% 50%)',
+
+    animationName: {
+      '0%': { transform: 'translateY(calc(var(--container-size) - var(--point-size)))' },
+      '60%, 100%': { transform: 'translateY(0)' },
+    },
+    animationDuration: '2s',
+    animationIterationCount: 'infinite',
+    animationFillMode: 'forwards',
+  },
+
+  graph: {
+    gridArea: 'graph',
+    position: 'relative',
+  },
+  graphX: {
+    color: tokens.colorNeutralStroke1,
+    fontFamily: tokens.fontFamilyMonospace,
+    position: 'absolute',
+    left: '10px',
+    top: '10px',
+  },
+  graphT: {
+    color: tokens.colorNeutralStroke1,
+    fontFamily: tokens.fontFamilyMonospace,
+    position: 'absolute',
+    bottom: '10px',
+    right: '10px',
+  },
+
+  duration: {
+    placeSelf: 'center',
+
+    width: '128px',
+    height: '4px',
+    margin: '64px auto',
+    backgroundColor: tokens.colorNeutralForeground3,
+
+    animationName: {
+      from: { transform: 'rotate(0)' },
+      to: { transform: 'rotate(180deg)' },
+    },
+    animationIterationCount: 'infinite',
+  },
+  svg: {
+    borderLeft: `1px solid ${tokens.colorNeutralStroke1}`,
+    borderBottom: `1px solid ${tokens.colorNeutralStroke1}`,
+
```

Comment:
The graphs are not clipped:
<img width="276" alt="image" src="https://github.com/user-attachments/assets/1bc236de-86dc-480c-8916-6edd728b115a">

```suggestion

    overflow: 'visible'

```